index.ts

```diff
@@ -377,6 +377,12 @@ collect.set('isRepoMainSettings', [
 	'https://github.com/sindresorhus/refined-github/settings',
 ]);
 
+export const isUserSettings = (url: URL | HTMLAnchorElement | Location = location): boolean => url.pathname.startsWith('/settings/');
+collect.set('isUserSettings', [
+	'https://github.com/settings/',
```

Comment:
This doesn't exist and should be removed.

Also the tests are failing because all the URLs in this file are tested against each other, so if any other URL matches this function, it should be included in this array. The test error will tell you exactly which URLs are missing.

Reply:
Sorry my fault, I was in rush and only made the change without running the tests locally first. I made the change in 229cc84

Reply:
No worries, some repos have stricter tests than others 😅